README.md

```diff
@@ -104,15 +104,15 @@ Everything that can be checked from the command-line version of the tool can be
 
 ### Mac
 
-The CLI tool for cfn-guard is available via [homebrew](https://brew.sh/).
+The CLI tool for cfn-guard is available via [homebrew](https://formulae.brew.sh/formula/cloudformation-guard).
 
 Installation via homebrew:
 ```
 brew install cloudformation-guard
 ```
 
 ### Windows
-The CLI tool for cfn-guard is available via [chocolatey](https://chocolatey.org).
+The CLI tool for cfn-guard is available via [chocolatey](https://chocolatey.org/packages/cloudformation-guard/1.0.0).
```

Comment:
```suggestion
The CLI tool for cfn-guard is available via [chocolatey](https://chocolatey.org/packages/cloudformation-guard).
```

Reply:
fair. i was unsure if this would appear since the package isn't indexed on chocolatey yet and is still under review. will change.